src/GroupProject/Views/BankId/shared/_bankid.cshtml

```diff
@@ -5,67 +5,63 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@ViewData["Title"]</title>
-    <link rel="stylesheet" type="text/css" href="~/bankid/Css/bid.css">
-    <link rel="stylesheet" type="text/css" href="~/bankid/Css/header.css">
-    <link rel="stylesheet" type="text/css" href="~/bankid/Css/main.css">
     <environment names="Development">
-
+        <link rel="stylesheet" type="text/css" href="/bankid/Css/bid.css">
+        <link rel="stylesheet" type="text/css" href="/bankid/Css/main.css">
+        <script src="http://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
     </environment>
     <environment names="Staging,Production">
-        <!-- CDN links here see https://docs.asp.net/en/latest/client-side/bootstrap.html -->
+        <script src="http://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
+        <link rel="stylesheet" type="text/css" href="~/bankid/Css/bid.css">
+        <link rel="stylesheet" type="text/css" href="~/bankid/Css/main.css">
     </environment>
```

Comment:
når linkene er like for alle enviroments kan vi likegodt sette de direkte inn i filen